tests/functional/limits/kvdb_limits_test.c

```diff
@@ -0,0 +1,218 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Can you separate this out into individual files? I can at least see a `cursor_limits_test`, `transaction_limits_test`, `kvdb_limits_test`, and `kvs_limits_test`

Reply:
Hmm, there are very few test cases here to split across multiple files. I think I'll leave it as is for now, and if we add more cases in the future, we can split them out.